visual-tests/scripts/visual-tests-build.mjs

```diff
@@ -0,0 +1,72 @@
+import path from 'path';
+import execa from 'execa';
+import fse from 'fs-extra';
+
+// Main wrapper should be defined as a first one -
+// it will be used to generate golden screenshots,
+// rest of wrappers will be compared to it.
+const wrappers = ['js', 'angular-13', 'react'];
+// const wrappers = ['js', 'angular-13', 'react', 'vue'];
+const baseBranch = 'develop';
+const currentBranchName = (await execa.command('git rev-parse --abbrev-ref HEAD', { silent: true })).stdout;
+const dirs = {
+  init: process.cwd(),
+  examples: '../examples/next/visual-tests',
+  codeToRun: 'demo',
+};
+
+// const HOTversion = '12.2.0';
+
+process.chdir(dirs.examples);
+execa.command('npm install');
+
+// If we are on a base branch, we do not want to run all of tests
+// and make screenshots for all of wrappers.
+// On base branch we need to create golden screenshots from main wrapper only,
+// which is declared as a first position in wrappers array.
+
+for (let i = 0, maxi = (currentBranchName === baseBranch ? 1 : wrappers.length); i < maxi; ++i) {
+  process.chdir(`${wrappers[i]}/${dirs.codeToRun}`);
+
+  // eslint-disable-next-line no-await-in-loop
+  await execa.command('npm install', { stdout: 'inherit' });
```

Comment:
Just a suggestion - if the script installs and builds multiple examples, we could install them in bulk using the `install` command of their "sub-workspaces".

For example, instead of running `npm install` for each of the examples in `next/visual-tests/vue`, we could run it once, just in the `next/visual-tests/vue` directory. All the shared dependencies will be installed once, and they'll be reused between the examples.

Reply:
@maciejbo-handsontable, what about that?

Reply:
if I undertood this suggestion correctly, then it's already done - moved from e.g. `vue/demo` just to `vue`